lib/exandra/connection.ex

```diff
@@ -671,8 +680,36 @@ defmodule Exandra.Connection do
     do: raise(ArgumentError, "constraints are not supported by Exandra")
 
   @impl Ecto.Adapters.SQL.Connection
-  def execute_ddl({_command, %Index{}, _}),
-    do: raise(ArgumentError, "indexes are not supported by Exandra")
+  def execute_ddl({command, %Index{} = index}) when command in [:create, :create_if_not_exists] do
+    unless is_nil(index.prefix),
+      do: raise(ArgumentError, "indexes with prefixes are not supported by Exandra")
```

Comment:
I think we might want to raise if folks use the `:concurrently` option too. It's in the examples in the docs for `Ecto.Migration.index/3`, so I think it might slip up 🙃 

```suggestion
    unless is_nil(index.prefix),
      do: raise(ArgumentError, "indexes with prefixes are not supported by Exandra")
      
    if index.concurrently do
      raise ArgumentError, "concurrent index creation is not supported by Exandra"
    end
```